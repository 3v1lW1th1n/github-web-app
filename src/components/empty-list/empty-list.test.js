import 'jest-styled-components'

import React from 'react'
import { shallow, mount } from 'enzyme'
import EmptyList from './'

describe('<EmptList />', () => {
  it('should match default snapshot', () => {
    const component = shallow(<EmptyList>Nenhum item encontrado</EmptyList>)
    expect(component).toMatchSnapshot()
  })
  it('should render the children', () => {
    const component = mount(<EmptyList>Nenhum item encontrado</EmptyList>)
    expect(component.text()).toEqual('Nenhum item encontrado')
  })

  it('should have the right displayName', () => {
    expect(EmptyList.displayName).toEqual('EmptyList')
  })
})
