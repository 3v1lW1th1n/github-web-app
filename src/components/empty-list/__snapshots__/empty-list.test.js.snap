// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptList /> should match default snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "EmptyList-sc-1f6m2ig-0",
        "isStatic": true,
        "rules": Array [
          "padding:48px;margin:0 auto;text-align:center;",
        ],
      },
      "displayName": "EmptyList",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "EmptyList-sc-1f6m2ig-0",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Nenhum item encontrado
</StyledComponent>
`;
